.idea/misc.xml

```diff
@@ -3,7 +3,7 @@
   <component name="ProjectKey">
     <option name="state" value="project://e2804f05-5315-4fc6-a121-c522a6c26470" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="16" project-jdk-type="JavaSDK">
```

Comment:
Why specify JDK_15 as languageLevel?

Reply:
Like maven above, this is an IDE setting. It would be better to define this in maven, then anyone with a different IDE could use the Java project without the need to set it in their IDE. And this change is not specifying JDK_15, it's actually removing it. Look at the file, I'm now using `languageLevel="JDK_1_8"`